primes.rb

```diff
@@ -1,6 +1,42 @@
+# THE EASY (AND FAST) WAY:
+
+require 'prime'
+
 class Primes
   def self.sum_to(limit = 100)
     # TODO - add your prime number solution here...
-    "I'm working on it!"
+    sum = 0
+    Prime.each(limit) do |prime|
```

Comment:
Since `Prime.each` returns an enumerable you should be able to use one of `reduce`, `collect`, `inject` or friends to perform this sum. See what you can achieve :wink: